WireDomain/Sources/WireDomain/Notifications/Components/Legacy/SyncEventsStep.swift

```diff
@@ -53,15 +54,32 @@ final class SyncEventsStep: Component<SyncEventsDependency>, SyncEventsStepProto
 
     private var selfClientID: String
 
+    private let pushChannelMonitor: PushChannelMonitor
+
     init(
         parent: any Scope,
         selfClientID: String
     ) {
         self.selfClientID = selfClientID
+        self.pushChannelMonitor = PushChannelMonitor(
+            clientID: selfClientID,
+            postingNotificationName: DarwinNotification
+                .releasingPushChannelAccess,
+            observingNotificationName: DarwinNotification
+                .requestingPushChannelAccess
+        )
         super.init(parent: parent)
     }
 
+    private var currentTask: Task<Void, any Error>?
```

Comment:
this task is used to embed the sync in NSE and to be cancelled if mainApp request access to the websocket

Reply:
so i try but failed to make it work